protoarray/src/test/java/tech/pegasys/artemis/protoarray/ComputeDeltasTest.java

```diff
@@ -0,0 +1,277 @@
+/*
+ * Copyright 2020 ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package tech.pegasys.artemis.protoarray;
+
+import static com.google.common.primitives.UnsignedLong.ZERO;
+import static com.google.common.primitives.UnsignedLong.valueOf;
+import static org.assertj.core.api.Assertions.assertThat;
+import static tech.pegasys.artemis.protoarray.HashUtil.getHash;
+import static tech.pegasys.artemis.protoarray.ProtoArrayForkChoice.computeDeltas;
+
+import com.google.common.primitives.UnsignedLong;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import org.apache.tuweni.bytes.Bytes32;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+public class ComputeDeltasTest {
```

Comment:
I did not understand this to be honest: 

> ProtoArrayForkChoiceComputeDeltasTest if we do want to separate test classes for the one production class 

Nevertheless, I switched to `ProtoArrayForkChoiceTest `, since that makes sense.